fre/app/mask_atmos_plevel.py

```diff
@@ -49,61 +51,59 @@ def maskAtmosPlevel_subtool(infile, outfile, psfile):
 
     # Write the output file if anything was done
     if ds_out.variables:
-        print(f"Modifying variables '{list(ds_out.variables)}', appending into new file '{outfile}'")
+        print(f"Modifying variables: {list(ds_out.variables)}\n appending into new file {outfile}")
         write_dataset(ds_out, ds_in, outfile)
     else:
-        print(f"No variables modified, so not writing output file '{outfile}'")
-    return None
+        print(f"No variables modified, so not writing output file {outfile}")
+
 
 
 def preprocess(ds):
     """add needs_atmos_masking attribute if var ends with _unmsk"""
-
     for var in list(ds.variables):
         if var.endswith('_unmsk'):
             ds[var].attrs['needs_atmos_masking'] = True
-
     return ds
 
 
 def mask_field_above_surface_pressure(ds, var, ds_ps):
     """mask data with pressure larger than surface pressure"""
-
     plev = pressure_coordinate(ds, var)
 
     # broadcast pressure coordinate and surface pressure to
     # the dimensions of the variable to mask
     plev_extended, _ = xr.broadcast(plev, ds[var])
     ps_extended, _ = xr.broadcast(ds_ps["ps"], ds[var])
+
     # masking do not need looping
     masked = xr.where(plev_extended > ps_extended, 1.0e20, ds[var])
+
     # copy attributes, but it doesn't include the missing values
     attrs = ds[var].attrs.copy()
+
     # add the missing values back
     attrs['missing_value'] = 1.0e20
     attrs['_FillValue'] = 1.0e20
     masked.attrs = attrs
+
     # transpose dims like the original array
     masked = masked.transpose(*ds[var].dims)
-
     print(f"Processed {var}")
 
     return masked
 
 
-def pressure_coordinate(ds, varname, verbose=False):
+def pressure_coordinate(ds, varname):#, verbose=False):
```

Comment:
`verbose` wasn't used, I see

Reply:
true! commented out... since it's not a BAD idea